pipeline/risteys_pipeline/finregistry/load_data.py

```diff
@@ -0,0 +1,56 @@
+"""Functions for loading the FinRegistry datasets"""
+
+import pandas as pd
+from risteys_pipeline.config import *
+from risteys_pipeline.log import logger
+from risteys_pipeline.finregistry.preprocess_data import (
+    preprocess_minimal_phenotype_data,
+    preprocess_wide_first_events_data,
+    preprocess_endpoints_data,
+)
+
+
+def load_minimal_phenotype_data(
+    data_path=FINREGISTRY_MINIMAL_PHENOTYPE_DATA_PATH, preprocess=False
+):
+    """Loads minimal phenotype data as a dataframe and optionally performs preprocessing."""
+    logger.info("Loading minimal phenotype data")
+    cols = ["FINREGISTRYID", "date_of_birth", "death_date", "sex"]
+    df = pd.read_csv(data_path, usecols=cols, header=0, sep=",")
```

Comment:
Are `header` and `sep` needed here?

Reply:
Not needed because they're the defaults (or the default of `header` is "infer") but I prefer them :)